include/polynomial.h

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#include <cmath>
+#include <array>
+#include <vector>
+
+// Implemented according to https://en.wikipedia.org/wiki/Cubic_equation
+
+namespace quda
+{
+
+  inline std::vector<double> quadratic_formula(std::array<double, 3> coeff)
+  {
+
+    std::vector<double> z;
```

Comment:
Why not just write this as `std::vector<double> z(2)`?

Reply:
See the response to the following comment.